### DISCUSSION TITLE
Chromium language select

### DISCUSSION THREAD

**Discussion by roelvdberg** [gotenberg/gotenberg#899] [Q&A] [answered]
I am trying to convert a website to PDF, the website has configured the language as NL.
When I open the PDF in and look at the meta data there is no language configured.

I changed the docker image to include this: [switch-language](https://gotenberg.dev/docs/configuration#switch-language) but that didn't change anything.

Is it possible to set the language with chromium?

**Comment by gulien** [comment, marked as answer]
Hello @roelvdberg,

> I changed the docker image to include this: [switch-language](https://gotenberg.dev/docs/configuration#switch-language) but that didn't change anything.I changed the docker image to include this: [switch-language](https://gotenberg.dev/docs/configuration#switch-language) but that didn't change anything.

Indeed, it's because it's related to LibreOffice.

AFAIK, Chromium does not set this metadata by itself, nor does it offer a way to set it 🤔
